backend/ml/mehestan/primitives.py

```diff
@@ -83,8 +83,9 @@ def QrUnc(
         qr_med = QrMed(W, w, x, delta)
     qr_dev = QrDev(W, default_dev, w, x, delta, qr_med=qr_med)
     delta_2 = delta ** 2
+    bound = np.inf if W <= 0 else 1 / W
     h = W + np.sum(
-        w * np.minimum(1, delta_2 * (delta_2 + (x - qr_med) ** 2) ** (-3 / 2))
+        w * np.minimum(bound, delta_2 * (delta_2 + (x - qr_med) ** 2) ** (-3 / 2))
     )
```

Comment:
Due to the definition of "Collaborative preference scaling factor of non-scaling-calibration contributors", `QrUnc` may be called with `W = 0`. 
For instance, this is the case for entities where all individual scores are zero. The uncertainty about the scaling factor is computed using $W = 8W||\theta_n||\_\infty$ where $||\theta_n||\_\infty = 0$

@lenhoanglnh What do you think? Is it correct to extend the definition of `QrUnc` to W=0, or should this case be handle explicitly in the scaling algorithm?